docs/HOL_Continuous-Deployment-Using-Custom-Agent/README.md

```diff
@@ -0,0 +1,121 @@
+# HOL - Parts Unlimited MRP App Continuous Deployment with Visual Studio Team Services - Local Agent #
+
+In this lab, you will learn how to deploy the Parts Unlimited MRP App in an automated fashion onto a local agent on a Linux VM. After this lab, you will have a working, automated build in Visual Studio Team Services that will build, test, and deploy the Parts Unlimited MRP app to a Virtual Machine in Azure.
+
+>**Note:** If you would like to trigger continuous deployments using the hosted VSTS agent instead of a local agent, see [this lab](https://github.com/Microsoft/PartsUnlimitedMRP/tree/master/docs/HOL_Continuous-Deployment).  
+
+
+###Pre-requisites###
+
+- Completion of the Continuous Integration with Parts Unlimited MRP HOL
+- An active Azure Subscription
+- An active Visual Studio Team Services Account
+
```

Comment:
An Personal Authentication Token setuped : (+ Hyperlink to the vsts website with all the steps to create one)